src/components/editTestCase/EditTestCase.tsx

```diff
@@ -212,11 +210,7 @@ const EditTestCase = () => {
   const { updateMeasure } = measureStore;
   const load = useRef(0);
   const [canEdit, setCanEdit] = useState<boolean>(
-    measure?.createdBy === userName ||
-      measure?.acls?.some(
-        (acl) =>
-          acl.userId === userName && acl.roles.indexOf("SHARED_WITH") >= 0
-      )
+    checkUserCanEdit(measure?.createdBy, measure?.acls)
```

Comment:
this logic seems incorrect...shouldn't we be checking that the currently logged in user has edit permissions? This appears to be checking only if the measure creator has edit permissions.

Reply:
@nmorasb Yes we are checking the currently logged in user has edit permissions. That part of logic is done in madie-util, which takes the logged in user from okta, and does the comparison with the passed in createdBy or acls.